### PR TITLE
Docu: telemetry.enabled in config.sample.php 

### PR DESCRIPTION
**Pull request by jnweiger**
Document the system config variable telemetry.enabled in config.sample.php
Needed for a clean config to docs run.

### CODE REVIEW

**Comment by phil-davis** [comment]
The title of the PR says it should be added to https://github.com/owncloud/core/blob/master/config/config.apps.sample.php

But it is added to https://github.com/owncloud/core/blob/master/config/config.sample.php

Is this a core setting, or an app setting?

**Review by mmattel — APPROVED** [review]
LGTM 👍 
Must be backported to 10.15
Imho this is a core setting and the title is wrong...

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/core/40020/3/2
needs a valid commit title

File: config/config.sample.php [hunk 1]
```diff
@@ -1882,4 +1882,15 @@
...
+/**
+ * Enable Sending Telemetry Reports
+ * Telemetry data is a subset of the config report as produced by the command `occ configreport:generate`.
+ *
+ * * If set to true, a daily telemetry report is sent to https://telemetry.owncloud.com/oc10-telemetry
+ * * If set to false, no telemetry reports are sent.
+ *
+ * Default: true
+ */
+'telemetry.enabled' => true,
```
Comment (iasdeoupxe) [inline config/config.sample.php:1895]: Should default to `false`, not `true ` :-1:

Comment (jnweiger) [reply config/config.sample.php:1895]: The intended logic is default enabled, and opt out. Sorry.

Comment (iasdeoupxe) [reply config/config.sample.php:1895]: Already discussed here: https://github.com/owncloud/configreport/pull/197#discussion_r1667255519

**Pull request merged**